src/shims/env.rs

```diff
@@ -22,18 +22,6 @@ impl EnvVars {
             }
         }
     }
-
-    pub(crate) fn get(&self, name: &[u8]) -> Option<&Pointer<Tag>> {
-        self.map.get(name)
-    }
-
-    pub(crate) fn unset(&mut self, name: &[u8]) -> Option<Pointer<Tag>> {
-        self.map.remove(name)
-    }
-
-    pub(crate) fn set(&mut self, name: Vec<u8>, ptr: Pointer<Tag>) -> Option<Pointer<Tag>>{
-        self.map.insert(name, ptr)
-    }
 }
 
 pub(crate) fn alloc_env_value<'mir, 'tcx>(
```

Comment:
Does this really still need to be public to the entire crate?

Reply:
No it does not, I'm correcting it now